### DISCUSSION TITLE
In-class Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#36] [Assignments]
Respond to ANY or all of the prompts below in 2-3 sentences:

### 4. How do you seek to be peer and/or mentor for civil society?

You do not need to respond to any students.

**Comment by NishuKrish** [comment]
It was a little difficult avoiding giving advice during this in-class exercise, as in such situations, it is usually expected for someone to give explicit advice. However, by asking questions, I felt that my partner was able to discover problems on their own, as just by explaining their project, they were able to discover things that could be improved upon. I feel like this technique is really valuable as peer/mentor for civil society, as this is one way in which projects can be improved upon in order to better suit the needs of the community.

**Comment by ananyakodali17** [comment]
I think that stating my problem and having more questions asked about the problem forced me to really think about the issue myself, and the questions helped me gain a new perspective on my dilemma. Giving advice is a default when people come to me with a problem, so it was a little difficult to refrain from giving advice.

**Comment by daikis12** [comment]
It was difficult to avoid giving advice because when somebody tells me that he is struggling with something, I'd naturally assume that this person is seeking advice from me. Even if just listening to his problem may be enough, and he may not expect a piece of advice, it is hard not to give my opinion about his particular issue, unless I am completely clueless.

**Comment by kris10fu** [comment]
I think that talking through my group's problem helped me understand more of the specific details that we were struggling with because I needed to communicate what the actual issue was. After talking with my partner, it did lead me to have some different ideas on what we could do to address the problem. It was hard to not give advice at first because that's usually what we provide when it comes to peer feedback, but it did make me have to think about their problem from different angles.

**Comment by codingdianat** [comment]
Sharing my problem out loud made me realize that the other person was also having a similar problem as me and we were able to brainstorm ideas to address our shared problems.  When we spoke about our problem we realized that it has a similar root cause and as a result we talked about ways to address it.

**Comment by stefuntran** [comment]
I seek to be a mentor for civil society by actively joining student organizations that reach out to high school students and even underclassmen at UCLA. Mentoring could be about the college application process to navigating cultural and societal expectations.  Learning from my mistakes and experiences, I strive to be that person I needed when I was younger.

**Comment by angelineeow** [comment]
I think it was difficult to avoid giving advice and asking questions to them. I realized I always have been trying to solve the problems for myself and others. I brought my attention to actually listening to others instead of giving solutions that might not be helpful for the person.

**Comment by emmashanson** [comment]
I found this exercise to be helpful since rather than expecting to hear advice from my peer, I knew that I would receive follow up questions about my problem and this made me critically think about the problem I had and make some discoveries about some potential solutions too.  I think in general, it is hard to give advice but in a context like this where the goal was to provide additional questions for my peer to think about, it seemed easier for both sides.

**Comment by kaittli** [comment]
Sharing my problem to my partner was insightful mainly because voicing my thoughts out loud gives me more clarity about how I feel about my own situation. When things are trapped inside my head, my thoughts get very muddled together, and it's difficult for me to identify what my stance exactly is, if I have one. I believe that talking to others leads you to feel more comfortable with the situation and that sharing with others can lead to possible solutions.

**Comment by lindseymardona** [comment]
Stating my problem didn't really bring any insights but it felt nice to acknowledge them. My partner was moreso empathetic rather than bringing up things I hadn't considered. It wasn't hard to avoid giving advice, in fact it was easier to just ask questions to understand better since you're not trying to fix their problem, you're just talking it out. Talking to other people and asking questions can help people discover new perspectives without shoving suggestions or your own POV in their faces.

**Comment by bittertang** [comment]
It was difficult to avoid offering solutions to my partner, but I realized that simply listening can also help the other person. For me, I was able to think about my own problem from a different perspective. Overall, it made me think about the intentions of our own project, which is to simply provide a platform to allow community members to share stories.

**Comment by ntran27** [comment]
I didn't really find it difficult to not give advice. This is because outside of the class context I usually learn by listening, asking questions, and gathering information first before offering my own advice or thoughts. Also most of the work environments I've been in have involved peers, advisors, and supervisors who prioritize autonomy and agency so from them I've learned the importance of being careful with giving out unsolicited advice.

**Comment by XiongHer** [comment]
Stating the problem out loud is pretty helpful because it allows me to hear my thoughts and rethink if they make sense or not. Sometimes, having a problem in mind sounds good but when I say it out loud, it doesn’t make sense at all. Also, having a peer to listen and provide feedback helps me think from new perspectives and how to approach the problem from different angles.

**Comment by antonyblin** [comment]
Talking about my problem out loud actually helped me to better conceptualize it and understand how I might create solutions for myself. My partner did not bring up too much that I had not considered, but I feel that that was reasonable in light of the problem I presented. It was not really difficult to avoid giving advice because I typically do not like to present answers and assistance flat out without some prodding. I seek to be a peer for civil society by being there to listen to community members and peers and offer solicited assistance as necessary.

**Comment by kericsp** [comment]
I think that stating our problems out loud helped to refocus the project on the goal we had in mind when we started out, and to rethink the problems as simply stages of the process rather than insurmountable roadblocks, such as having troubles getting in touch with the nonprofits we reached out to. My partner was helpful in reminding me that the problems I faced weren't impossible to solve, it just takes time.

**Comment by willa-21** [comment]
Personally, peer mentorship has always been critical to my own growth and learning. As a peer and mentor, I hope to ask questions and provide encouragement that allows others to think critically, thoroughly, and equitably --about civil society as a whole. I think having the opportunity to create relationships and spaces where we are able to both support and challenge each other is vital to growth and development.

**Comment by ashley-lu** [comment]
Stating the problem out loud really helped to put into words what we needed to improve on, and my partner was able to ask questions about our issue that were insightful and led me to think about our project differently. It was definitely hard to give advice, as I feel it's something I'm used to doing when talkin with a friend or peer.

**Comment by shubhgupta2018** [comment]
When speaking about the problem out loud, I realized how a lot of my struggles as a student and critical cartographer lie in the fact that I rely on a stream of consciousness conversation to fully understand what I am discussing instead of utilizing concise language that is easily understood. This is especially important for me because I want my work within this class and beyond to be palatable for audiences of varying backgrounds, and perhaps working on the means by which I communicate problems, solutions, and ideas related to social injustices will be most helpful in the future.

**Comment by joonyloony** [comment]
I think that speaking my project and the steps we have taken so far out loud to someone who knows nothing about it definitely made me more conscious of how to present my project and in a way, help me catch certain problems with my presentation of my project. I think that it's really good to speak to an outside perspective sometimes in order to step away from the project and evaluate it more objectively

**Comment by spolcharoen** [comment]
Stating our group’s problem made it clear that we have to think through the user journey of our site better. Although it was difficult to think of a follow up question rather than give advice, it opened the door to more interesting conversation about both our projects. My partner and I also bonded by talking a bit about the data clubs we are in on campus! I think that reaching out in your organizations would be an easy way to seek a peer in this space.

**Comment by hajovel** [comment]
When simply stating the problem out loud, it did help in bringing insights. It helped me realize what in particular I was not understanding or had come to misunderstand, and sensing the error in my own statement. Talking it over with my group was very helpful as they also pointed out factors I had not considered. I hope from this experience I can focus on finding a balance between peer and mentor, and as a mentor for civil society, push for ethics and safety of others. As a peer, I would want to learn these ethics and experiences from others, to see where I can better improve myself based on my own lived experiences.
